libs/application/templates/parental-leave/src/fields/DraftRequiresAction/index.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { FC } from 'react'
+
+import { useLocale } from '@island.is/localization'
+
+import { FieldBaseProps, MessageFormatter } from '@island.is/application/core'
+import { Box, Button, Text, toast } from '@island.is/island-ui/core'
+import { FieldDescription } from '@island.is/shared/form-fields'
+
+import { parentalLeaveFormMessages } from '../../lib/messages'
+
+import { SUBMIT_APPLICATION } from '@island.is/application/graphql'
+import { useMutation } from '@apollo/client'
+
+function handleError(error: string, formatMessage: MessageFormatter): void {
+  toast.error(
+    formatMessage(
+      {
+        id: 'application.system:submit.error',
```

Comment:
Would be nice to move it to `libs/application/core/src/lib/messages.ts` and import directly from there